maturity_test.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/PuerkitoBio/goquery"
+)
+
+var (
+	githubApiAuthorizationToken = os.Getenv("GITHUB_API_TOKEN")
```

Comment:
```suggestion
	githubAPIAuthorizationToken = os.Getenv("GITHUB_API_TOKEN")
```

Or to respect the GitHub syntax 


```suggestion
	apiTokenGitHub = os.Getenv("GITHUB_API_TOKEN")
```